draft-ietf-rats-corim.md

```diff
@@ -1466,6 +1466,10 @@ CoRIM-based data structures define an external representation of Conceptual Mess
 Appraisal processing describes both mapping transformations and Verifier reconciliation ({{sec-verifier-rec}}).
 Non-CoRIM-based data structures require mapping transformation, but these are out of scope for this document.
 
+If a Verifier requires CoBOMs, then the CoRIM inputs indicated explicitly by the CoBOM are the only appropriate and required CoRIMs.
+If the Verifier does not require CoBOMs, then the CoRIM inputs consist of the knowledge base of CoRIMs available to the Verifier at the beginning of appraisal.
+The knowledge base state SHOULD have an immutable label for attestation results to have auditable provenance.
```

Comment:
I like this a lot. However, it's a requirement that affects the tracking done by the ACS to record the tag-id of the processed triples.

Reply:
The entire attestation appraisal is frozen to the input state. The construction of the attestation context should be deterministic. The appraisal of the context should be deterministic. The result construction should be free to reference the label for the input state. I don't think we need to change ECT or ACS data types.